app/controllers/ConfirmSubmissionController.scala

```diff
@@ -181,7 +180,7 @@ class ConfirmSubmissionController @Inject()(mandationStatusCheck: MandationStatu
 
           vatReturnsService.nrsSubmission(periodKey, htmlPayload, sha256Checksum, id, receipt) flatMap {
             case Left(ErrorModel(BAD_REQUEST, _)) =>
-              logger.debug(s"[ConfirmSubmissionController][submitToNRS] - NRS returned BAD_REQUEST: ${BAD_REQUEST.toString}")
```

Comment:
same here

Reply:
i think this should also be kept as errorlog since the user is re-directed to an internal server error page